.claude/skills/finance-assess/templates/report.html

```
@@ -0,0 +1,827 @@
+<!-- Title is the artifact's name in the gallery and must not change between
+     runs. It deliberately carries no count: how many months survive the
+     coverage gate is a figure, and the scope row below reports it. -->
+<title>The Trustworthy Months</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)